src/Linter/Sniff/ConfigNoCacheSniff.php

```diff
@@ -0,0 +1,15 @@
+<?php declare(strict_types=1);
+namespace Helmich\TypoScriptLint\Linter\Sniff;
+
+use Helmich\TypoScriptLint\Linter\Sniff\Visitor\ConfigNoCacheVisitor;
+use Helmich\TypoScriptLint\Linter\Sniff\Visitor\SniffVisitor;
+
+class ConfigNoCacheSniff extends AbstractSyntaxTreeSniff {
```

Comment:
Could you also adjust the `README.md` to mention this new sniff?

Reply:
Will do